.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
           make "${KATA_ASSET}-tarball"
           build_dir=$(readlink -f build)
           # store-artifact does not work with symlink
-          sudo cp -r "${build_dir}" "kata-build"
+          mkdir -p kata-build && cp "${build_dir}/kata-static-${KATA_ASSET}.tar.*" kata-build/.
```

Comment:
Will this skip the extra modules/headers in the kernel build now? Maybe `"${build_dir}"/kata-static-${KATA_ASSET}*.tar.*` would be better (note the moving of double quotes too that I think Wainer said was needed for the wildcard matches to work